plugins/external/discover.go

```diff
@@ -4,7 +4,7 @@ import (
 	"os"
```

Comment:
@StewartJingga How about moving this external package to core and naming it plugin. Since this package is about providing core functionality of supporting a plugin system in meteor but it is not really a plugin. Is that correct understanding or am I missing something?

Reply:
im thinking core package is something that is fully consist of core system and logic without any interface contexts and super minimum libraries (e.g. http, rpc, cli, statsd etc).

if you check current packages inside `core` package, they are mostly plain native golang struct with minimum library or package. (it only uses yaml library)

right now external plugin has a heavy rpc context and `go-plugin` library and putting them inside core package would tear down the scope of core package.

the other reason I put external plugins to plugins folder is because I'm guessing when people are looking at our code, whenever they are looking for external plugins that would be the first folder they go into.

wdyt? @ravisuhag

Reply:
Also atm, I do not see plugins folder as the a mere folder to store all the plugins, I see it as a plugins package which contains means to fetch our internal plugins as well as external plugins.

Reply:
sounds good!